.appveyor.yml

```diff
@@ -102,7 +96,7 @@ test_script:
   #
   # Also use -oldStyle flag which uses an alternate instrumentation hook that is available for .NET Core
   # Some comments on this can be found here: https://github.com/OpenCover/opencover/issues/601
-  - .\tools\OpenCover.4.6.519\tools\OpenCover.Console.exe -target:dotnet.exe -targetargs:"test --logger \"trx;LogFileName=netstandard_test_results.xml\" .\Keen.NetStandard.Test\Keen.NetStandard.Test.csproj /property:DebugType=Full" -mergeoutput -output:coverage.xml -filter:"+[*]* -[*.Test]*" -oldStyle -returntargetcode -register:user
+  - .\tools\OpenCover.4.6.519\tools\OpenCover.Console.exe -target:dotnet.exe -targetargs:"test --logger \"trx;LogFileName=netstandard_test_results.xml\" .\Keen.NetStandard.Test\Keen.NetStandard.Test.csproj /property:DebugType=Full" -mergeoutput -output:coverage.xml -filter:"+[*]* -[*.Test]* -[Moq*]*" -oldStyle -returntargetcode -register:user
 
   # Upload merged coverage results from both test runs
```

Comment:
This comment is outdated.

Reply:
Ha, I was looking at OpenCover [Issue 601](https://github.com/OpenCover/opencover/issues/601) thinking you meant the `-oldStyle` flag wasn't needed anymore, but then I realized you were referring to the "from both test runs" part :satisfied:, will fix.